docs/networks/how_to_backup_and_recover_your_node.md

```diff
@@ -1,46 +1,64 @@
-# How To Backup And Recover Your Node
+# Backup And Recover Your validator
 
-It is pretty straight forward to setup a node when your old node when your old setup doesn't work anymore. You can simply
-run
+We made it simple and straight forward to recover a node with its validators when your old node doesn't work anymore or you want to transfer everything to a new machine.
 
-```bash
-lukso network init --chain l16
-```
+## The keystore
 
-in another instance and the setup is done in no time.
-
-The problem starts with the keystore. How do you copy and move the keystore to another instance.
-
-The keystore is determinstic derivative of 4 values
+The keystore consists the following 4 values:
 
```

Comment:
That is not true, the keystore is a derivative of these 4 values. The keystore itself is a directory full of keys ;)